include/variables.h

```diff
@@ -559,83 +216,25 @@ extern volatile OSTime gRSPAudioTotalTime;
 extern volatile OSTime gRSPGFXTotalTime;
 extern volatile OSTime gRSPOtherTotalTime;
 extern volatile OSTime gRDPTotalTime;
-extern SpeedMeterTimeEntry* gSpeedMeterTimeEntryPtr;
-//extern ? D_8016A590;
-//extern ? D_8016A594;
-//extern ? D_8016A598;
-//extern ? D_8016A5A0;
-//extern ? D_8016A5B8;
-//extern ? D_8016A5C4;
-//extern ? D_8016A5D0;
-//extern ? D_8016A5E8;
-//extern ? D_8016A5F8;
-//extern ? D_8016A608;
-//extern ? D_8016A618;
-//extern ? D_8016A628;
-//extern ? D_8016A638;
-//extern ? D_8016A688;
-//extern ? D_8016A698;
-//extern ? D_8016A6A8;
-//extern ? D_8016A6E8;
-//extern ? D_8016A6F8;
-//extern ? D_8016A704;
-//extern ? D_8016A710;
-//extern ? D_8016A720;
-//extern ? D_8016A724;
-//extern ? D_8016A728;
-//extern ? D_8016A730;
-//extern ? D_8016A732;
-//extern ? D_8016A734;
-//extern ? D_8016A736;
-//extern ? D_8016A738;
-//extern ? D_8016A73A;
-//extern ? D_8016A73C;
-//extern ? D_8016A73E;
-//extern ? D_8016A740;
-//extern ? D_8016A750;
-//extern ? D_8016A794;
-//extern ? D_8016A7AC;
 extern FaultThreadStruct gFaultStruct;
+
+// data/code_800F7260.bss.s
 extern ActiveSound gActiveSounds[7][MAX_CHANNELS_PER_BANK]; // total size = 0xA8
 extern u8 gSoundBankMuted[];
 extern u16 gAudioSfxSwapSource[10];
 extern u16 gAudioSfxSwapTarget[10];
 extern u8 gAudioSfxSwapMode[10];
-//extern ? D_8016E320;
-//extern ? D_8016E348;
+extern u8 D_8016E348[4];
+extern u32 sAudioSeqCmds[0x100];
```

Comment:
What's the reason to have the externs here instead of inside the file like it was?
If the variables aren't global then they shouldn't be extern'd in here imo, but I might be missing something

Reply:
it's weird because `code_800F9280` has the only references to some variables which are in the bss of another file, `code_800F7260`

How do you define if a variable is "global" or not? If it's a variable from one file used in another file, then this fits, but maybe that's not the definition we go with?

I could move most of the variables from `code_800F7260` bss to z64audio.h, and many externs to specific headers, but that feels like wasted efforts as long as we haven't split and included the headers properly for each source file

Reply:
Yes, this applies to nearly all audio variables that are currently in `variables.h` and it currently still needs some work. Nearly all audio variables are either localized to a single audio file (no extern needed) or are localized to the audio subsystem (and should be kept to that scope instead of being made global). It'll be worth making a separate PR to address that.

`D_8016E348` and `sAudioSeqCmds` specifically are in `code_800F7260.bss.s` which has not been imported yet, but both are localized to `code_800F7260`. So these externs could be localized to `code_800F7260` and the externs will be removed once that bss file is imported. I'm fine addressing this now or saving it for when the audio externs are properly addressed. Either way, this will have to be removed from here eventually.

Reply:
Yes actually, I'll add that I mixed up the files and these bss variables are localized to `code_800F9280`, even though they are present in `code_800F7260.bss`. That's fine, and strange ordering of bss/data variable is common in audio, so externs between audio files for variables is also common.

Reply:
so just leave this as it currently stands in the pr and someone will clean it up when moving more of the audio data?

Reply:
Right now a variable is considered global (and named accordingly) if it's used or meant to be used in other files. And ideally, all global variables (and only global variables) should be exposed in `variables.h` until we restructure headers, especially for bss because where the externs are will matter for -g3 bss ordering so it will need some work.

So for these audio variables, I think we should have externs in the files themselves, basically wherever the variables would be once migrated to C, and the externs for global variables in `variables.h` if there are any.

Ideally we should clean this up right now, but I guess we can also work on a follow up PR for that if it's easier

Reply:
Agreed that if it's not addressed here, then it should be addressed very soon, before we introduce -g3 bss reordering via retail